### PR TITLE
tests for clearTimeout and clearInterval to clear any type of timer

### LINKED ISSUE #2277: Add a test for the interchangeability of clearTimeout and clearInterval
Both in window and worker scope. See also https://github.com/whatwg/html/pull/282.

### PR DESCRIPTION
**Pull request by frehner**
Fixes #2277

This is my first JS test, so feel free to correct any mistakes or bad assumptions I made. 

Thank you!

### CODE REVIEW

File: html/webappapis/timers/cleartimeout-clearinterval.any.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    }),
+    0
+  );
+
+  clearInterval(handle);
+
+  setTimeout(() => {
+    t.done();
+  }, 100);
+});
```
Comment (jdm) [inline html/webappapis/timers/cleartimeout-clearinterval.any.js:14]:
```suggestion
}, "Clear timeout with clearInterval");
```

File: html/webappapis/timers/cleartimeout-clearinterval.any.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    }),
+    0
+  );
+
+  clearTimeout(handle);
+
+  setTimeout(() => {
+    t.done();
+  }, 100);
+});
```
Comment (jdm) [inline html/webappapis/timers/cleartimeout-clearinterval.any.js:29]:
```suggestion
}, "Clear interval with clearTimeout");
```

**Review by jdm — CHANGES_REQUESTED** [review]
This looks good with the addition of test names! Thanks for fixing this!

**Comment by frehner** [comment]
whoops! 

updated with the names now. thanks

**Review by jdm — APPROVED** [review]

**Pull request merged**